cli/scale.go

```diff
@@ -91,7 +91,12 @@ func runScale(args *docopt.Args, client *controller.Client) error {
 		if err != nil {
 			fmt.Println(commands["scale"].usage)
 		}
-		processes[arg[:i]] = val
+		processType := arg[:i]
+		if _, ok := release.Processes[processType]; ok {
+			processes[processType] = val
+		} else {
+			fmt.Printf("No process type exists with name \"%s\".\n", processType)
```

Comment:
I think a case can be made for either solution. I suppose it's safer to go your route though.